src/mlpack/tests/image_load_test.cpp

```diff
@@ -67,6 +67,63 @@ BOOST_AUTO_TEST_CASE(SaveImageAPITest)
     BOOST_REQUIRE_EQUAL(im1[i], im2[i]);
 }
 
+/**
+ * Test if the image is saved correctly using API for transpose.
+ */
+BOOST_AUTO_TEST_CASE(SaveImageTransposeAPITest)
```

Comment:
Thanks for looking into the API, I guess it would be useful to test if we can load/save ``arma::mat`` as well.